src/main/resources/application.yml

```diff
@@ -4,9 +4,11 @@ server:
 ---
 spring:
   profiles.active: local
+  applicationURL: localhost:9000

```

Comment:
Instead of setting this property as just a default like we do profile, add an entry in the local profile (not yet in this file):

spring:
  profiles: local
    applicationURL: localhost:9000

Also - doesn't the application get hosted on port 8080? Not 9000?